src/components/EntityInfo/EntityMenu.vue

```diff
@@ -94,6 +74,59 @@
         Manage the digital credentials generated for the business.
       </v-tooltip>
     </span>
+
+    <!-- More Actions -->
+    <span v-if="isBusiness && !isHistorical">
+      <v-menu offset-y transition="slide-y-transition" v-model="expand">
+        <template v-slot:activator="{ on }">
+          <v-btn text color="primary" class="menu-btn pr-3" v-on="on">
+            <v-icon v-if="expand">mdi-menu-up</v-icon>
+            <v-icon v-else>mdi-menu-down</v-icon>
+            <span>More Actions</span>
+          </v-btn>
+        </template>
+
+        <v-list dense>
+          <v-list-item-group color="primary">
+            <!-- Dissolve Business -->
+            <v-tooltip right content-class="right-tooltip">
+              <template v-slot:activator="{ on }">
+                <v-list-item
+                  id="dissolution-list-item"
+                  v-on="on"
+                  @click="promptDissolve()"
+                  :disabled="!isAllowed(AllowableActions.VOLUNTARY_DISSOLUTION)"
+                >
+                  <v-list-item-title>
+                    <span class="app-blue">Dissolve this Business</span>
+                  </v-list-item-title>
+                </v-list-item>
+              </template>
+              Dissolving the business will make this business historical
+              and it will be struck from the corporate registry.
+            </v-tooltip>
+
+            <!-- Consent to Continue Out -->
+            <v-tooltip right content-class="right-tooltip">
+              <template v-slot:activator="{ on }">
+                <v-list-item
+                  v-if="isBenBcCccUlc || isCoop"
+                  id="cco-list-item"
+                  v-on="on"
+                  @click="goToConsentContinuationOutFiling()"
+                  :disabled="!isAllowed(AllowableActions.CONSENT_CONTINUATION_OUT)"
+                >
+                  <v-list-item-title>
+                    <span class="app-blue">Consent to Continue Out</span>
+                  </v-list-item-title>
+                </v-list-item>
+              </template>
+              Submit a Consent to Continue Out of the province ofB.C.
```

Comment:
Typo. Also check with Yui on whether it should be "B.C." or "BC".

Reply:
Fixed typo. tooltip based on UXpin but will confirm with Yui